api-tester/src/defaults/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -85,6 +97,7 @@ public void onSuccess(@NonNull AmazonLWAConsentStatus contentStatus) {
         purchases.getCustomerInfo(receiveCustomerInfoListener);
         purchases.getCustomerInfo(CacheFetchPolicy.CACHED_OR_FETCHED, receiveCustomerInfoListener);
         purchases.getAmazonLWAConsentStatus(getAmazonLWAContentStatusCallback);
+        purchases.getCustomerCenterConfigData(getCustomerCenterConfigCallback);
```

Comment:
Related to a previous comment, it can be argued that this is not needed, as it isn't intended to be a public API.